guide/C/Makefile.am

```diff
@@ -15,6 +15,8 @@ entities = \
 	ch_basics.xml \
 	ch_accts.xml \
 	ch_txns.xml \
+	ch_importing.xml \
+	ch_configuring.xml \
```

Comment:
This looks fine.

There seems to be an issue with githubs GUI and my restricted understanding of it. On the Conversation/discussion tab and the diffs in the mails I get displayed the frozen state from my first comment.

So this part is resolved.